stories/ui/Rating.js

```diff
@@ -0,0 +1,138 @@
+/* @flow */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Rating from 'ui/components/Rating';
+import type { Props as RatingProps } from 'ui/components/Rating';
+
+import { createChapters } from '../utils';
+import Provider from '../setup/Provider';
+
+
+export type Props = {|
+  props: RatingProps,
+|};
+
+function createPropsMatrix(): Array<Props> {
+  return [
+    {
+      props: {
+        readOnly: false,
+        styleSize: 'small',
+        rating: 4,
+        yellowStars: true,
+      },
+    },
```

Comment:
Here are a few missing states:

```diff
diff --git a/stories/ui/Rating.js b/stories/ui/Rating.js
index 9321035de..af88c437a 100644
--- a/stories/ui/Rating.js
+++ b/stories/ui/Rating.js
@@ -15,6 +15,22 @@ export type Props = {|
 
 function createPropsMatrix(): Array<Props> {
   return [
+    {
+      props: {
+        readOnly: false,
+        styleSize: 'small',
+        rating: undefined,
+        yellowStars: true,
+      },
+    },
+    {
+      props: {
+        readOnly: false,
+        styleSize: 'small',
+        rating: null,
+        yellowStars: true,
+      },
+    },
     {
       props: {
         readOnly: false,
@@ -39,6 +55,14 @@ function createPropsMatrix(): Array<Props> {
         yellowStars: true,
       },
     },
+    {
+      props: {
+        readOnly: false,
+        styleSize: 'large',
+        rating: undefined,
+        yellowStars: true,
+      },
+    },
     {
       props: {
         readOnly: false,
@@ -73,6 +97,14 @@ function createPropsMatrix(): Array<Props> {
         yellowStars: true,
       },
     },
+    {
+      props: {
+        readOnly: true,
+        styleSize: 'small',
+        rating: undefined,
+        yellowStars: false,
+      },
+    },
     {
       props: {
         readOnly: true,
@@ -97,6 +129,14 @@ function createPropsMatrix(): Array<Props> {
         yellowStars: false,
       },
     },
+    {
+      props: {
+        readOnly: true,
+        styleSize: 'large',
+        rating: undefined,
+        yellowStars: false,
+      },
+    },
     {
       props: {
         readOnly: true,
@@ -106,6 +146,22 @@ function createPropsMatrix(): Array<Props> {
         className: 'RatingManager-UserRating',
       },
     },
+    {
+      props: {
+        readOnly: true,
+        styleSize: 'large',
+        rating: 3.4,
+        yellowStars: false,
+      },
+    },
+    {
+      props: {
+        readOnly: true,
+        styleSize: 'large',
+        rating: 4,
+        yellowStars: false,
+      },
+    },
   ];
 }
 
```

Reply:
I guess i didn't see these ones on the frontend so I left those out but I can definitely add